helm/boot.sh

```diff
@@ -1,19 +1,10 @@
 #!/usr/bin/env bash
-mkdir ~/.funcx
-mkdir ~/.funcx/$1
-mkdir ~/.funcx/credentials
+
+EP_NAME="$1"
+shift
+
+mkdir -p ~/.funcx/"$EP_NAME" ~/.funcx/credentials
 cp /funcx/config/config.py ~/.funcx
-cp /funcx/$1/* ~/.funcx/$1
-cp /funcx/credentials/storage.db ~/.funcx/
-if [ -z "$2" ]; then
-  funcx-endpoint start $1
-else
-  funcx-endpoint start $1 --endpoint-uuid $2
-fi
+cp /funcx/"$EP_NAME"/* ~/.funcx/"$EP_NAME"
 
-while pgrep funcx-endpoint >/dev/null;
-    do
-        echo "funcx-endpoint process is still alive. Next check in 600s."
-        sleep 600;
-    done
-echo "funcx-endpoint process exited. Restarting endpoint"
+exec funcx-endpoint start "$EP_NAME" "$@"
```

Comment:
Oof, that'll (re)teach me to go so fast.  Basically popped this out of my git stash and said "go!"  Yes, that does come with a matching update.  Let me regather my thoughts on why I made this change.

Reply:
This initially came about because I just wanted to specify the CLI args.  That's perhaps too strong, but now I've implemented:

- the `endpointUUID` **must** be specified
- optionally, the user may specify `endpointCLIargs` for any additional arguments

For example, to attach `--debug` to the _interchange_ (as opposed to `workerDebug`), can specify it as:
```helm
endpointCLIargs: "--debug"
```

Reply:
To wit, the `endpointUUID` can be any uuid the user wants, so long as it isn't already taken by another account.  So for this specification, one can either start an endpoint locally to collect that UUID, or just generate one.  For example, from the command line:
```shell
$ uuid -v4
94f41efb-e292-43ce-9cfe-65c0d68237dc
```
Alternatively, `uuid` is a built-in module in Python:
```python
>>> import uuid
>>> uuid.uuid4()
UUID('ed13a388-6caf-4024-8605-aae42ec26fc9')
```